test/integration/submission/migration_utils/schema_builder.ts

```diff
@@ -0,0 +1,142 @@
+import { migrationDiffs } from './stub-diffs';
+import _ from 'lodash';
+import fs from 'fs';
+import { schemaV1_accessible, schemaV2, schemaV3 } from './static-schema-data';
+const DICTIONARY_NAME = 'ARGO Clinical Submission';
+
+export const buildDynamicStubSchema = () => {
+  const convertToDictionaryNotation = (schemaVersion: Object) => {
+    return Object.values(schemaVersion).map(schema => {
+      const { fields, ...rest } = schema;
+      const newFields = Object.values(schema.fields);
+      return _.assign(rest, { fields: newFields });
+    });
+  };
+
+  const legacyDictionaries = [
+    {
+      schemas: convertToDictionaryNotation(schemaV1_accessible),
+      name: DICTIONARY_NAME,
+      version: '1.0',
+    },
+
+    {
+      schemas: schemaV2,
+      name: DICTIONARY_NAME,
+      version: '2.0',
+    },
+
+    {
+      schemas: schemaV3,
+      name: DICTIONARY_NAME,
+      version: '3.0',
+    },
+  ];
+
+  // Change 8.1.1.1 adding new enum value Adding an enum to the codelist
```

Comment:
lol I don't understand the numbering `8.x.x.x`, is this the wajiha decimal system? or am I missing something else.

Reply:
oh I see it these the are numbers from the spec, nvm

Reply:
come on you've never heard of the wajiha decimal system? 😡